public_website/src/ui/components/simulator/SliderField.tsx

```diff
@@ -49,6 +51,10 @@ export function SliderField({
     [onChange]
   )
 
+  useEffect(() => {
```

Comment:
Je ne suis pas sur de comprendre si tu as agit ici donc je pose quand même la question.
Pourquoi passer par un state et un useEffect ?

Reply:
Lors du chargement, il y avait une erreur d'hydratation à cause du Select. Tu as un autre solution ? Mais on peut isoler ce comportement pour éviter que tout le composant soit render.

Reply:
Je te mets le message d'erreur de Nextjs :
```
Unhandled Runtime Error
Error: Hydration failed because the initial UI does not match what was rendered on the server.
Warning: Expected server HTML to contain a matching <span> in <option>.
See more info here: https://nextjs.org/docs/messages/react-hydration-error
```

Reply:
Ok je comprends mieux merci :)